_posts/2023-06-11-ayb-helping-you-own-your-data.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: 'ayb: Helping you own your data'
+date: 2023-06-11 22:45 +0000
+---
+## Introduction
+
+Today's databases are ubiquitous, but frustratingly inaccessible to most people. Your own data likely lives in thousands of databases at various companies and organizations, but if you wanted to create a database for yourself in which to store data and share it, you need the skills of both a system administrator and a software engineer. By virtue of the complexity of database management systems (and market forces), your data either lives in other people's databases or outside of a database in text files and spreadsheets.
```

Comment:
The last sentence gets to the point

Reply:
But with trivially easy extraction and client scale beyond a node.

Reply:
I love the last sentence as well! I integrated a less branded version of it into the intro and walked back specific examples of file types to keep focus on the problem rather than the solution.